docs/guide/replacement-guides/tempy.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Modern alternatives to the tempy package for creating temporary files and directories
+---
+
+# Replacements for `tempy`
+
+## Node.js (since v14.x)
+
+Node.js has the [`fs.mkdtemp`](https://nodejs.org/api/fs.html#fsmkdtempprefix-options-callback) function for creating a unique temporary directory.
+
+Example:
+
+```js
+import {temporaryDirectory} from 'tempy' // [!code --]
+const tempDir = temporaryDirectory() // [!code --]
+
+import {mkdtemp, realpath} from 'node:fs/promises' // [!code ++]
+import {join} from 'node:path' // [!code ++]
+import {tmpdir} from 'node:os' // [!code ++]
+
+// macOS and some other platforms may return a symlink from `os.tmpdir`.
+// For some applications, this can cause problems; thus, we use `realpath`.
+const tempDir = await mkdtemp(join(await realpath(tmpdir()), 'foo-')) // [!code ++]
+```
+
+Directory cleanup can be done by passing `{recursive: true}` to [`fs.rm`](https://nodejs.org/api/fs.html#fsrmpath-options-callback), available in v14.14.0 and up:
+
+```js
```

Comment:
this section doesn't need to be a diff since we're not going from anything to anything

its just some info (that we can use `recursive: true`)